rolling/distribution.yaml

```diff
@@ -2263,6 +2263,13 @@ repositories:
       url: https://github.com/hatchbed/log_view.git
       version: ros2
     status: developed
+  marker_msgs:
```

Comment:
Would you like to include a `source` and `doc` entry? It seems there was one for the ROS 1 version.

https://github.com/ros/rosdistro/blob/bd05eb7efff3cfefee1934b89c0334cbac3866aa/jade/distribution.yaml#L2850-L2864

Reply:
Oh, same comment as here: https://github.com/ros/rosdistro/pull/36519#discussion_r1142214819